AsyncDisplayKit/ASVideoNode.mm

```diff
@@ -217,6 +220,21 @@ - (void)fetchData
       _player = [[AVPlayer alloc] initWithPlayerItem:_currentItem];
       _player.muted = _muted;
     }
+    __weak __typeof(self) weakSelf = self;
+    _timeObserver = [_player addPeriodicTimeObserverForInterval:CMTimeMake(1, 1) queue:NULL usingBlock:^(CMTime time){
```

Comment:
Hello @Xinchi, we can move it to delegate as @ejensen mentioned. How do you think would it be better to create whole observer from delegate or just asking for preferred seconds?